packages/flutter/test/rendering/editable_test.dart

```diff
@@ -2,11 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/foundation.dart';
```

Comment:
What are the changes in this file for? Maybe revert?

Reply:
Yes in the end the is no changes on that file as I moved my test, I'll revert :)